edc-extension4aas/src/main/java/de/fraunhofer/iosb/app/CustomAuthenticationRequestFilter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021 Fraunhofer IOSB, eine rechtlich nicht selbstaendige
+ * Einrichtung der Fraunhofer-Gesellschaft zur Foerderung der angewandten
+ * Forschung e.V.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.fraunhofer.iosb.app;
+
+import org.eclipse.dataspaceconnector.api.auth.AuthenticationRequestFilter;
+import org.eclipse.dataspaceconnector.api.auth.AuthenticationService;
+
+import de.fraunhofer.iosb.app.model.configuration.Configuration;
+import jakarta.ws.rs.container.ContainerRequestContext;
+
+/**
+ * Custom AuthenticationRequestFilter filtering requests that go directly to an
+ * AAS service (managed by this extension).
+ */
+public class CustomAuthenticationRequestFilter extends AuthenticationRequestFilter {
+
+    private final Configuration config;
+    private static final String selfDescriptionEndpointName = "selfDescription";
```

Comment:
For better maintenance we should centralize such static endpoint names. There are other places where this endpoint value is used e.g. https://github.com/FraunhoferIOSB/EDC-Extension-for-AAS/blob/main/edc-extension4aas/src/main/java/de/fraunhofer/iosb/app/Endpoint.java#L355